docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -1,22 +1,30 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    res = client.submit_qubo(obj_QUBO, target="toshiba_bifurcation_qpu")
</pre><p>Comment out this line. We want people to do <code>dry-run</code> by default. Also add a comment saying that <code>dry-run</code> runs on the simulator and if they want to run it on the machine they should delete the <code>dry-run</code> argument.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>in the markdown above, i've include the comment explaining what to do if you want to run it via dry-run or the toshiba. let me know if that was enough.</p>

Reply:
fixed it!